en/flight_modes/follow_me.md

```diff
@@ -8,16 +8,16 @@ author: Jimmy Johnson
 
 *Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.
 
-The vehicle will automatically yaw to face and follow the target from a specified [relative position](#NAV_FT_FS), [horizontal separation](#NAV_FT_DST) and [height](#NAV_MIN_FT_HT) above the home position.
+The vehicle will automatically yaw to face and follow the target from a specified [relative orientation](#NAV_FT_FS), [distance](#NAV_FT_DST) and [height](#NAV_MIN_FT_HT) (reference point : home, ground (with distance sensor), target's GPS altitude, depending on the [altitude control mode](#NAV_FT_ALT_M)).

```

Comment:
I'm not sure a reader will get the point about "reference point ; ..."

Reply:
Removed!